Foundation/URLSession/URLSession.swift

```diff
@@ -194,7 +194,8 @@ open class URLSession : NSObject {
     fileprivate var invalidated = false
     fileprivate static let registerProtocols: () = {
 	// TODO: We register all the native protocols here.
-        let _ = URLProtocol.registerClass(_HTTPURLProtocol.self)
+       _ = URLProtocol.registerClass(_HTTPURLProtocol.self)
```

Comment:
Should we remove the TODO here?

Reply:
I presume No .We have only implementation of HTTP and FTP in place as of now .Still we may need to extend the protocol support to FILE or DATA  or any other native protocols .Hence having this will help implementers to know  where to register the same ..